uiplib/scheduler.py

```diff
@@ -108,5 +108,3 @@ def run(self):
             self.change_random()
             self.setStartTime(time.time())
             self.changeCycle()
-        else:
```

Comment:
Because the download is threaded, when this line is run, the folder is empty. Which prints this false message